client/components/create-post.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+
+class CreatePost extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      postBody: '',
+      postTags: '',
+      post: null
+    };
+    this.goToViewPost = this.goToViewPost.bind(this);
+  }
+
+  goToViewPost(event) {
+    this.props.setView('viewPost', {});
+  }
+
+  handleChange(event) {
+    if (event.target.id === 'postBody') {
```

Comment:
I would suggest using 
```suggestion
    this.setState({[event.target.id]: event.target.value})
```
rather than if statements in this scenario.